rlberry/manager/evaluation.py

```diff
@@ -84,14 +89,114 @@ def evaluate_agents(agent_manager_list,
 
     return output
 
+def read_writer_data(agent_manager,
```

Comment:
I think that if `input_dir` is given, we don't actually need `agent_manager` as input. We could have a single argument for both, that is either:
* An instance of `AgentManager`
* A list of instances of `AgentManager`
* A directory (which contains all the info that we need to read the data)

Reply:
Ok, good idea.